app/src/sharedTest/java/org/oppia/app/recyclerview/RecyclerViewMatcher.kt

```diff
@@ -88,4 +94,23 @@ class RecyclerViewMatcher {
       )
     }
   }
+
+  private class RecyclerViewGridItemCountAssertion(private val count: Int) : ViewAssertion {
+
+    override fun check(view: View, noViewFoundException: NoMatchingViewException?) {
+      if (noViewFoundException != null) {
+        throw noViewFoundException
+      }
+
+      check(view is RecyclerView) { "The asserted view is not RecyclerView" }
+
+      checkNotNull(view.layoutManager) { "No layoutManager is assigned to RecyclerView" }
```

Comment:
Suggest instead: ``check(view.layoutManager is GridLayoutManager) { "RecyclerView must use GridLayoutManager" }``

This makes the cast below safer (in that a more useful error message will be shown in this situation), and the cast check implies non-nullness so a separate null check isn't needed.

Reply:
Ok .Done